projects/packages/account-protection/src/class-password-detection.php

```diff
@@ -68,6 +68,27 @@ public function login_form_password_detection( $user, string $password ) {
 		return $user;
 	}
 
+	/**
+	 * Redirect and exit.
+	 *
+	 * @param string $redirect_location The redirect location.
+	 *
+	 * @return never
+	 */
+	protected function redirect_and_exit( string $redirect_location ) {
```

Comment:
I added these functions because if you `exit;` in your code, you'll have a hard time running PHPUnit tests against it. This way, it can be disabled dynamically by the tests.